HeterogeneousCore/CUDAUtilities/test/assert_t.cu

```diff
@@ -0,0 +1,13 @@
+#include "HeterogeneousCore/CUDAUtilities/interface/cuda_assert.h"
+#include "HeterogeneousCore/CUDAUtilities/interface/requireCUDADevices.h"
+
+__global__ void testIt(int one) { assert(one == 1); }
```

Comment:
A comment explaining how this test works would be nice.

Reply:
> A comment explaining how this test works would be nice.

@fwyzard @VinInn Does this test really test anything else than using `assert()` with `#include ".../cuda_assert.h`" compiles and runs with and without `-DGPU_DEBUG`?

Reply:
no more than that, no.
Still need to be tested, IMHO.

Reply:
Thanks @VinInn, I'll add a comment.